packages/lit-ssr/src/lib/util/escaping.ts

```diff
@@ -0,0 +1,41 @@
+/**
```

Comment:
Maybe use https://www.npmjs.com/package/escape-html instead?

Reply:
Went with https://www.npmjs.com/package/@wordpress/escape-html instead, which has an `escapeAttribute` separate from `escapeHTML`.

Reply:
Actually, switched to suggested `escape-html` due to licensing. I believe it's safe to put html-escaped strings in attribute value position (it's a subset that needs to be escaped, but fine to over-escape).